### PR TITLE
Issue #1 - Release as npm package

### PR DESCRIPTION
**Pull request by armenzg**
In this change we remove all UI related components and prepare the package
to be released as an npm package.

### CODE REVIEW

**Comment by armenzg** [comment]
Hi @eliperelman this is the original work of @csd713 
We're dropping the UI related components and only shipping the library.

Is this good to release?

@csd713 I had to do some shenanigans locally since we have two different repos.

File: src/index.jsx [hunk 1]
```diff
@@ -1,22 +1 @@
...
-  ), root,
-);
-
-// This is needed for Hot Module Replacement
-if (module.hot) {
-  module.hot.accept('./App', load);
-}
-
-load();
+export default () => 'Welcome to perf-goggles';
```
Comment (eliperelman) [inline src/index.jsx]: It doesn't look like this library does anything.

File: package.json [hunk 2]
```diff
@@ -4,31 +4,15 @@
   "main": "index.js",
   "repository": "git@github.com:mozilla-frontend-infra/perf-goggles.git",
   "author": "Armen Zambrano G. <armenzg@gmail.com>",
-  "license": "MIT",
+  "license": "MPL-2.0",
```
Comment (eliperelman) [inline package.json]: I don't see a `main` entry here for what this library runs.

**Comment by armenzg** [comment]
@csd713 You can ignore my previous comment. I edited it.

File: package.json (hunk 2 above)
Comment (armenzg) [reply package.json]: Something like ["main": "build/index.js"](https://github.com/taskcluster/taskcluster-client-web/blob/master/package.json#L4)?:

File: src/index.jsx (hunk 1 above)
Comment (armenzg) [reply src/index.jsx]: Would including the perfherder.js module and exporting it be the right way?

Something like what you do [here](https://github.com/taskcluster/taskcluster-client-web/blob/master/src/index.js)?

**Comment by armenzg** [comment]
Hi @eliperelman , while @csd713 manages to get back to this I rebased this branch over master.
This rebase included testing and coverage.

For some odd reason, I need to install `node-fetch` as it might be needed by mocha, however, I would have assume that @neutrinojs/mocha getting installed with the Neutrino wizard should have taken care of it.
I've added `node-fetch` as a separate commit on this PR.

If I remove `node-fetch` I get this output:
```
armenzg@armenzg-mbp perf-goggles$ yarn coverage
yarn run v1.7.0
$ nyc -r lcov -r text yarn test
$ neutrino test
module.js:557
    throw err;
    ^

Error: Cannot find module 'node-fetch'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/armenzg/repos/perf-goggles/node_modules/fetch-mock/src/server.js:2:15)
    at Module._compile (module.js:660:30)
    at Module.replacementCompile (/Users/armenzg/repos/perf-goggles/node_modules/nyc/node_modules/append-transform/index.js:58:13)
    at module.exports (/Users/armenzg/repos/perf-goggles/node_modules/nyc/node_modules/default-require-extensions/js.js:8:9)
    at /Users/armenzg/repos/perf-goggles/node_modules/nyc/node_modules/append-transform/index.js:62:4
    at require.extensions.(anonymous function) (/Users/armenzg/repos/perf-goggles/node_modules/babel-register/lib/node.js:152:7)
    at Object.<anonymous> (/Users/armenzg/repos/perf-goggles/node_modules/nyc/node_modules/append-transform/index.js:62:4)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/armenzg/repos/perf-goggles/test/utils/perfherder.test.js:2:1)
    at Module._compile (module.js:660:30)
    at Module.replacementCompile (/Users/armenzg/repos/perf-goggles/node_modules/nyc/node_modules/append-transform/index.js:58:13)
    at loader (/Users/armenzg/repos/perf-goggles/node_modules/babel-register/lib/node.js:144:5)
    at require.extensions.(anonymous function) (/Users/armenzg/repos/perf-goggles/node_modules/babel-register/lib/node.js:154:7)
    at Object.<anonymous> (/Users/armenzg/repos/perf-goggles/node_modules/nyc/node_modules/append-transform/index.js:62:4)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at /Users/armenzg/repos/perf-goggles/node_modules/mocha/lib/mocha.js:231:27
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (/Users/armenzg/repos/perf-goggles/node_modules/mocha/lib/mocha.js:228:14)
    at Mocha.run (/Users/armenzg/repos/perf-goggles/node_modules/mocha/lib/mocha.js:536:10)
    at Object.<anonymous> (/Users/armenzg/repos/perf-goggles/node_modules/mocha/bin/_mocha:573:18)
    at Module._compile (module.js:660:30)
    at Module.replacementCompile (/Users/armenzg/repos/perf-goggles/node_modules/nyc/node_modules/append-transform/index.js:58:13)
    at module.exports (/Users/armenzg/repos/perf-goggles/node_modules/nyc/node_modules/default-require-extensions/js.js:8:9)
    at Object.<anonymous> (/Users/armenzg/repos/perf-goggles/node_modules/nyc/node_modules/append-transform/index.js:62:4)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Function.Module.runMain (module.js:701:10)
    at runMain (/Users/armenzg/.node-spawn-wrap-52168-ecf800a8f846/node:68:10)
    at Function.<anonymous> (/Users/armenzg/.node-spawn-wrap-52168-ecf800a8f846/node:171:5)
    at Object.<anonymous> (/Users/armenzg/repos/perf-goggles/node_modules/nyc/bin/wrap.js:23:4)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Function.Module.runMain (module.js:701:10)
    at /Users/armenzg/.node-spawn-wrap-52168-ecf800a8f846/node:178:8
    at Object.<anonymous> (/Users/armenzg/.node-spawn-wrap-52168-ecf800a8f846/node:181:3)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Function.Module.runMain (module.js:701:10)
    at startup (bootstrap_node.js:194:16)
    at bootstrap_node.js:618:3
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
----------|----------|----------|----------|----------|-------------------|
File      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
----------|----------|----------|----------|----------|-------------------|
All files |        0 |        0 |        0 |        0 |                   |
----------|----------|----------|----------|----------|-------------------|
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by eliperelman** [comment]
Neutrino would not include this for you. `fetch` is a browser API, and consuming it from Node.js means needing to polyfill it somehow.

**Comment by armenzg** [comment]
Thanks for the clarification!

@csd713 I've rebased your branch over one more change I've landed on master.
Please let me know when you will have time to finish this up.
I've been asked to tackle another project for the next week or so. I will be back after that 😄

**Comment by armenzg** [comment]
@eliperelman this is working for me locally with `npm link` and I tested publishing to verdaccio.

In short: I'm throwing away all UI components and only keeping the data manipulation part of it.

File: package.json [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
   "version": "1.0.0",
-  "main": "index.js",
-  "repository": "git@github.com:mozilla-frontend-infra/perf-goggles.git",
+  "main": "build/index.js",
+  "keywords": [
+    "mozilla",
+    "treeherder",
+    "perfherder"
+  ],
+  "repository": {
```
Comment (eliperelman) [inline package.json]: You can shorten this to:

```json
"repository": "mozilla-frontend-infra/perf-goggles"
```

File: package.json [hunk 4]
```diff
@@ -1,14 +1,20 @@
 {
   "name": "perf-goggles",
```
Comment (eliperelman) [inline package.json]: We now have a `@mozilla-frontend-infra` org on npm if you're interested.

**Review by eliperelman — APPROVED** [review]
